src/element/index.ts

```diff
@@ -85,3 +85,62 @@ export const getNonDeletedElements = (elements: readonly ExcalidrawElement[]) =>
 export const isNonDeletedElement = <T extends ExcalidrawElement>(
   element: T,
 ): element is NonDeleted<T> => !element.isDeleted;
+
+export const getElementsAfterReconcilation = (
+  currentElements: readonly ExcalidrawElement[],
+  sceneElements: readonly ExcalidrawElement[],
+  elementState: {
+    editingElement: NonDeletedExcalidrawElement | null;
+    resizingElement: NonDeletedExcalidrawElement | null;
```

Comment:
I'd just slap `appState` in here. It'll be obvious where this is coming, and not much reason to seemingly decouple this from appState because it's not.